rust/repo/{{ cookiecutter.repo_name }}/.github/workflows/ci.yaml

```
@@ -30,6 +31,76 @@ jobs:
       - uses: DavidAnson/markdownlint-cli2-action@v9
         with:
           globs: "**/*.md"
+      # TODO: Fix automatically
+
+  crev:
+    runs-on: ubuntu-22.04
+    needs:
+      - "rustfmt"
+      - "markdown-lint"
+    # Override errors here, eventually we'd like to start failing PRs based on this
+    continue-on-error: true
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: actions/cache@v3
+        continue-on-error: false
+        with:
+          path: |
+            Cargo.lock
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
+          restore-keys: ${{ runner.os }}-cargo-
+      - run: cargo crev trust --level high --no-commit https://github.com/mobilecoinfoundation/crev-proofs
```

:wrench: should this be installing crev?
```suggestion
      - run: |
        cargo install cargo-crev || true
        cargo crev trust --level high --no-commit https://github.com/mobilecoinfoundation/crev-proofs
```

---

Good catch, ty